### ISSUE TITLE
Improved documentation for new Flask developers

### ISSUE THREAD

**Issue by tonydalziel** [tonydalziel/spike#9]
Additional information should be provided at the bottom of the README file explaining how new Flask developers can run the API on their own machines. This will help attract more developers and help complete the project's documentation.

**Comment by xanderread** [comment]
I agree after I have completed basic endpoints of the api i will build some basic documentation
